simpleclient/src/main/java/io/prometheus/client/CollectorRegistry.java

```diff
@@ -130,6 +130,13 @@ public void clear() {
     return new MetricFamilySamplesEnumeration();
   }
 
+  /**
+   * Enumeration of metrics matching the specified names.
+   * <p>
+   * Note that the set of names will be matched against the metric sample names
```

Comment:
Time series name and metric name is the appropriate naming.

Reply:
Tweaked comment based on suggestion.